js/form.js

```diff
@@ -2,9 +2,19 @@ const form = document.querySelector('form');
 const emailError = document.querySelector('.form-error');
 
 form.addEventListener('submit', (e) => {
+  const name = form.name.value;
   const email = form.email.value;
+  const message = form.message.value;
   if (email !== email.toLowerCase()) {
     e.preventDefault();
     emailError.innerHTML = `<p class="error">Email should be written in lowercase letters: ${email}</p>`;
+  } else {
+    emailError.innerHTML = '';
+    const data = JSON.stringify({ name, email, message });
+    localStorage.setItem('data', data);
   }
 });
+const userData = JSON.parse(localStorage.getItem('data'));
+form.name.value = userData.name;
+form.email.value = userData.email;
+form.message.value = userData.message;
```

Comment:
- [ ] When the user changes the content of any input field, the data is saved to the local storage. kindly implement it as it is not saving
### Image
![ss](https://user-images.githubusercontent.com/81727988/180469380-5915f247-3616-4103-9a52-ed0153611408.png)

Reply:
it is saving on my end i just tested it again

Reply:
the information you put in this image is incorrect, as if a non email is inputed the submit form would not work such that the local storage doesnt get any data @roshan-bajgain